### ISSUE TITLE
Can we train the model using the prediction model?

### ISSUE THREAD

**Issue by BearVic** [fizyr/keras-retinanet#608]
Just out of curiosity, as title, can we train directly using the prediction model, so that we can extend the model based on the results from the prediction model?

**Comment by hgaiser** [comment]
Yeah you can. The inference model is simply the train model with some additional layers to compute boxes and scores and labels correctly.

To resume training from an inference model, use the `--weights` argument of `train.py`.

**Comment by BearVic** [comment]
Thanks! I have successfully built up a model based on the inference model. But new issues showed when backpropagation. It said gradient needed(something like K.sum K.round will cause this problem). So I have to train the whole model separately. Is there any solution can fix it or it has to train separately?
